docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure.md

```diff
@@ -65,53 +65,118 @@ showAsideToc: true
 
 </ul>
 
-
 This page details how to configure an Azure cloud provider for YugabyteDB clusters using the Yugabyte Platform console.
 
-
 ## Prerequisites
 
-You need to provide your cloud provider credentials - Subscription ID, Tenant ID, Application - ClientID/ClientSecret, Resource Group, Virtual Private Network on the Yugabyte Platform console. Yugabyte Platform uses the credentials to automatically provision and deprovision YugabyteDB instances.
-You will be able to see all the resources managed by Yugabyte Platform in your resource group, which will include - Virtual Machines, Network Interface, Network Security Groups, Public IP addresses, and Disks  
+You need to add the following cloud provider credentials in the Yugabyte Platform console:
 
-## Configure Azure
+* Subscription ID
+* Tenant ID
+* Application Client ID and Secret
+* Resource Group
 
+Yugabyte Platform uses the credentials to automatically provision and deprovision YugabyteDB instances.
+
+You'll be able to see all the resources managed by Yugabyte Platform in your resource group, which will include Virtual Machines, Network Interface, Network Security Groups, Public IP addresses, and Disks.
+
+## Configure Azure
 
 ![Prepare Azure cloud to install Yugabyte Platform](/images/yb-platform/install/azure/platform-azure-prepare-cloud-env-4.png)
 
-#### Provider name
-This is an internal tag used for organizing your providers, so you know where you want to deploy your YugabyteDB universes.
+**Provider name** is an internal tag used for organizing your providers, so you know where you want to deploy your YugabyteDB universes.
+
+**Subscription ID**. You need an Azure subscription ID for cost management. The virtual machine resources managed by the Platform will be tagged with this subscription.
+
+**Resource group** is the group in which YugabyteDB nodes compute and network resources will be created. Your Azure active directory application (clientId and client secret) needs to have `Network Contributor` and `Virtual Machine Contributor` roles assigned for this resource group.
+
+**Tenant ID** is the Azure Active Directory tenant ID which belongs to an active subscription. To find your tenant ID, follow the instructions on [this Microsoft Azure page](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#get-tenant-and-app-id-values-for-signing-in).
+
+**Client ID** is the client ID of an application registered in your Azure Active Directory.
```

Comment:
I know it's the same destination as above, but there's another section to generate/find the client ID/secret that could also be linked here. https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret